Standards/scs-0114-v1-secure-connections.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Secure Connections
+type: Standard  # | Decision Record | Procedural
+status: Draft
+track: IaaS  # | IaaS | Ops | KaaS | IAM
+---
+
+## Introduction
+
+A lot of internal and external connectivity is established to and within a cloud infrastructure.
+Due to the nature of the IaaS approach, many communication channels may occasionally or even primarily carry potentially sensitive data of customers.
+To protect this data from both tampering and unintended disclosure, communication channels need to be properly secured.
+
+For this reason, [SCS](https://scs.community) standardizes the use of common protection mechanisms for communication channels in OpenStack infrastructures.
+
+## Motivation
+
+As mentioned above, a lot of communication channels in an OpenStack infrastructure carry data that is potentially sensitive.
+For example this includes authentication data of customers and internal OpenStack configuration data such as encryption keys among others.
+OpenStack does not generically mandate or preconfigure the use of specific protection mechanisms by itself and instead only makes recommendations about best practices in its offical [Security Guide](https://docs.openstack.org/security-guide/).
+
+To address the potential lack of implementation of such mechanisms by a CSP and to establish a reliable foundation for communication data protection in SCS clouds, SCS formulates this standard for securing communication channels in the infrastructure, so that a customer can rely on adequate security mechanisms being in use.
```

Comment:
```suggestion
To address the potential lack of implementation of such mechanisms by a Cloud Service Provider and to establish a reliable foundation for communication data protection in SCS clouds, SCS project formulates this standard for securing communication channels in the infrastructure, so that a customer can rely on adequate security mechanisms being in use.
```